pages/common/mozillavpn.md

```diff
@@ -0,0 +1,36 @@
+# mozillavpn
+
+> A Virtual Private Network from the makers of Firefox.
+> More information: <https://github.com/mozilla-mobile/mozilla-vpn-client/wiki/Command-line-interface>.
+
+- Log in with interactive prompt:
+
+`mozillavpn login`
+
+- Connect to Mozilla VPN:
+
+`mozillavpn activate`
```

Comment:
Wondering if the VPN get activated for the current shell or the whole internet session (whatever goes out of the NIC) ?
Should we document that as well ?

Reply:
For the whole system. I guess it's not easy to setup a proxy just for the current bash session